### ISSUE TITLE
Should we need a "System settings"?

### ISSUE THREAD

**Issue by damo-da** [ServerChef/serverchef-ui#4]

**Comment by damo-da** [comment]
I guess whenever we hover over the "shutdown" icon on top right, we could add a system settings tab. We need this tools like gotty which are virtualhost-independent.

**Comment by Aagat** [comment]
Yep, we can have another menu item under that menu that allows users to adjust system settings.

For project independent tools, I was thinking about having unity dash like thing on top menu bar that can use used to quick launch both project related and independent things. Should help with user experience as well as making it easier to use with keyboard.
